samples/snippets/detect_intent_synthesize_tts_response.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2020 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
I think this should either

A) not be changed (e.g. keep this as `# Copyright 2020 Google LLC`)
B) the year should be added not replace (e.g. `# Copyright 2020-2022 Google LLC`)

Reference: https://opensource.google/documentation/reference/copyright#the_year